packages/extension/src/RichTextActions/RichTextActionsImpl.tsx

```diff
@@ -495,4 +496,44 @@ Do not converse with the user.
 
     this.context?.setIsLocked(false);
   }
+
+  public async insertContentStudioContent(): Promise<void> {
+    try {
+      const { proseMirrorEditorView, params, language } = this.context!;
+      const { dispatch } = proseMirrorEditorView;
+      let { state } = proseMirrorEditorView;
+
+      const studio = new AmplienceContentStudio({
+        baseUrl:
+          params?.tools?.contentStudio?.baseUrl ||
+          "https://app.amplience.net/content-studio",
+      });
+
+      const content = await studio.getContent();
+      const textFields = Object.keys(content.data)
```

Comment:
Can you use Object.values() here instead?

Reply:
yup, should make it easier to read - good spot!